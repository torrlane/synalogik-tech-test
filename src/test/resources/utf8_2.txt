Russian:

  From a Unicode conference invitation:

  Зарегистрируйтесь сейчас на Десятую Международную Конференцию по