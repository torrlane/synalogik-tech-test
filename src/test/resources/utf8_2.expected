Word count = 6
Average word length = 6.5
Number of words of length 1 is 1
Number of words of length 4 is 1
Number of words of length 7 is 2
Number of words of length 10 is 2
The most frequently occurring word length is 2, for word lengths of 7 & 10
